Source/SwiftLintFramework/Rules/Lint/UseCaseExposedFunctionsRule.swift

```diff
@@ -0,0 +1,126 @@
+import SourceKittenFramework
```

Comment:
Can you use [SwiftSyntax](https://github.com/apple/swift-syntax) instead of [SourceKitten](https://github.com/jpsim/SourceKitten)?

Apple maintains SwiftSyntax.

See: 

https://github.com/realm/SwiftLint/pull/4526
https://github.com/realm/SwiftLint/pull/4474
https://github.com/realm/SwiftLint/pull/4522
https://github.com/realm/SwiftLint/pull/4496
https://github.com/realm/SwiftLint/pull/4491

Reply:
> Can you use [SwiftSyntax](https://github.com/apple/swift-syntax) instead of [SourceKitten](https://github.com/jpsim/SourceKitten)?
> 
> Apple maintains SwiftSyntax.
> 
> See:
> 
> #4526 #4474 #4522 #4496 #4491

The SourceKitten API does look nicer in my opinion. We could try writing the next rule (callAsFunction) using SwiftSyntax. What do you think?

Reply:
@esilverberg CallAsFuntion rule written with SwiftSyntax